src/App.css

```diff
@@ -1,38 +1,231 @@
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
+form button {
+  width:auto;
+  align-self: flex-end;
+  background-color: rgb(12, 112, 12);
+  font-family: 'Marck Script', cursive;
+  color:  rgba(250, 235, 215, 0.8);
+  border: none;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.397);
+  margin-top: 1%;
+  padding:1%;
+  letter-spacing: 4px;
+  transition: 0.5s;
+  border-radius: 5%;
+  cursor: pointer;
+}
+
+form button:hover, form button:focus {
+  background-color:  rgba(250, 235, 215, 0.95);
+  color: rgb(12, 112, 12);
+}
+
+.menuContent {
+  width: 50%;
+  max-height: 0;
+  overflow: hidden;
+  text-align: center;
+  background-color: rgba(250, 235, 215, 0.9);
+  border-radius: 10%;
+}
+
+.collapsibleMenu {
+  z-index: 10;
+  margin-top: 1%;
+  display:flex;
   flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  align-items: flex-start;
+  width:100%;
 }
 
-.App-link {
-  color: #61dafb;
+
+.collapsibleMenu label {
+  font-family: 'Marck Script', cursive;
+  color: rgb(12, 112, 12);
+  font-weight: 600;
+  font-size: 1rem;
+  display: block;
+  cursor: pointer;
+  opacity: 0.8;
+  text-align: center;
+  padding-bottom: 1%;
+  width:50%;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
+.collapsibleMenu label i {
+  margin-right: 10px;
+}
+
+.collapsibleMenu input {
+  display: none;
+}
+
+.collapsibleMenu .menuContent ul {
+  padding: 0;
+  list-style: none;
+}
+.collapsibleMenu .entryButton {
+  color: black;
+  font-family: 'Marck Script', cursive;
+  line-height: 2;
+  text-decoration: underline dotted;
+  cursor: pointer;
+  background: none;
+  border: none;
+  transition: 0.3s;
+}
+
+.collapsibleMenu .deleteButton {
+  border: none;
+  padding: 5px;
+  font-size: 15px;
+  height: 28px;
+  width: 28px;
+  box-shadow: 0 2px 4px rgb(0, 0, 0);
+  cursor: pointer;
+  transition: all 0.3s ease;
+  margin: 3% 4% 0% 4%;
+  border-radius: 50%;
+  margin-bottom: 2%;
+  background-color: rgb(12, 112, 12);
+  color:  rgba(250, 235, 215, 0.8);
+}
+.collapsibleMenu .deleteButton:hover, .collapsibleMenu .deleteButton:focus {
+  background-color:  rgba(250, 235, 215, 0.95);
+  color: rgb(12, 112, 12);
+}
+
+
+.collapsibleMenu .menuButton {
+  background-color: rgb(12, 112, 12);
+  font-family: 'Marck Script', cursive;
+  color:  rgba(250, 235, 215, 0.8);
+  border: none;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.397);
+  margin-bottom: 1%;
+  padding:1%;
+  cursor: pointer;
+  letter-spacing: 4px;
+  transition: 0.5s;
+  border-radius: 5%;
+}
+
+.collapsibleMenu .menuButton:hover, .collapsibleMenu .menuButton:focus {
+  background-color:  rgba(250, 235, 215, 0.95);
+  color: rgb(12, 112, 12);
+}
+
+/* Toggle Effect */
+input:checked ~ .menuContent {
+  max-height: 100%;
+}
+
+.selectedWrapper {
+  margin-top: 25px;
+}
+
+.selectedWrapper h2 {
+  font-family: 'Marck Script', cursive;
```

Comment:
Not required but if you want to, you can use CSS custom properties. Define them in the root at the top of your style file and then reference them with the var CSS function. 

Example:
```
:root {
  --main-font: 'Marck Script', cursive;
}
```

And then where ever you want to use it do so like this:

```
element {
  font-family: var(--main-font);
}
```